src/hashTable.js

```diff
@@ -0,0 +1,60 @@
+export default class HashTable {
+  constructor() {
+    this.storage = []
+    this._length = 0
+    this.max = 10
+  }
+
+  put = (key, value) => {
+    if(typeof(key) === "undefined"){
+      throw("Cannot insert with undefined key!")
+    }
+    const hashIndex = this.hash(key, this.max)
+    this.storage[hashIndex] = value
+    this._length += 1
+  }
+
+  get = (key) => {
+    const hashIndex = this.hash(key, this.max)
+    return this.storage[hashIndex]
+  }
+
+  contains = (key) => {
+    if(this.get(key)){
+      return true
+    }
+    return false
+  }
+
+  iterate = passedInFunction => {
+    for (let i = 0; i < this.max; i++){
+      if(this.storage[i] !== undefined){
+        //unhash key - create unhash function?
```

Comment:
If you store the key with the value, you wouldn't need to "unhash". 😄  I would create a `Node` object for the `HashTable` as well, so that you can use [separate chaining](http://algs4.cs.princeton.edu/34hash/) to manage key collisions.